src/pages/Popup/index.html

```diff
@@ -6,6 +6,6 @@
   </head>
 
   <body>
-    <div id="app-container"></div>
+      <div id="app-container"></div>
```

Comment:
The indent should be 2 spaces. I don't know why this file changed.

Reply:
I found that `yarn run prettier` will not optimize the format of files with html as the suffix. 

This indentation issue was left by me and I've now fixed it. Previously relied too heavily on `yarn run prettier`.